### DISCUSSION TITLE
Recommendations for antibodies for NHP FFPE tissues

### DISCUSSION THREAD

**Discussion by rosedever** [IBEXImagingCommunity/ibex_imaging_knowledge_base#145] [Q&A] [answered]
Hello,
 
I am working on a panel for multiplexed imaging of NHP FFPE lung and liver samples. I found these antibodies for human lung and liver through the [OMAP initiative](https://humanatlas.io/omap):
- ARG1: Cell Signaling Technology 93668S
- ALB: Abcam ab240330; clone: EPR12774
- ABCA3: Novus Biologicals NBP1-89310
 
Would these work in my tissue?
 
I am interested in antibodies for labeling endothelial cells in NHP FFPE tissues. Do you know why CD31 (clone WM59) is less effective on FFPE sections than other methods of preparing tissues?

**Comment by radtkea** [comment, marked as answer]
Thank you for your question! NHP FFPE is an exciting frontier. 

Glad you are using the OMAP initiative to find liver and lung specific targets. I am not sure if these antibodies with work in NHP. Please see my comments below.

ARG1: Cell Signaling Technology 93668S: This antibody is validated for human tissues (vendor supplied images) and has reported reactivity for Human (H), Mouse (M), and Rat (R) but not Monkey (Mk). I would not recommend buying/testing this antibody in NHP tissues. 
ALB: Abcam ab240330; clone: EPR12774 Same thing with this antibody. No reported reactivity for monkey. 
ABCA3: Novus Biologicals NBP1-89310 Only reported for human reactivity.

Endothelial markers:
My colleagues have validated this [CD31 antibody for NHP FFPE tissues](https://www.abcam.com/products/primary-antibodies/cd31-antibody-ep3095-ab134168.html). It's available in many different conjugates from Abcam. 
This [CD34 antibody](https://www.thermofisher.com/antibody/product/CD34-Antibody-clone-QBEND-10-Monoclonal/MA1-10202) has been reported reactivity in NHP. We have used it for [human FFPE tissues](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD34_PE/0000-0003-4379-8967.html).
We report this [LYVE-1 antibody](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/LYVE-1_Unconjugated/0000-0001-9561-4256.html) in NHP FFPE tissues.

Here is a **GREAT** resource for antibodies validated for [CODEX imaging of NHP FFPE samples](https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2021.729845/full). The authors report what works and does not work based on IHC validation! It doesn't look like they have any endothelial markers. 

This paper on [IMC imaging of NHP FFPE samples](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9334813/) might also be useful for you as you build out panels. No endothelial markers.

One final thought is the use of a [lectin](https://vectorlabs.com/products/unconjugated-gsl-i-isolectin-b4) to label endothelial cells.  @colinchu72 has used [lectins](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/PNA_AF488/0000-0003-2088-8310.html) for IBEX imaging of the [human retina](https://cdn.humanatlas.io/hra-releases/v1.4/docs/omap/omap-12-eye-retina-ibex.html). 

Hope this helps! Happy to chat more and would love to help you share your knowledge with the community, when ready. You can grab the latest [reagent_resources.csv](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/blob/main/data/reagent_resources.csv) file from GitHub. Follow this template when recording your antibodies. Let me know if I can introduce you to any contacts with antibody suppliers. They are always looking for people to validate and report reactivity in NHP samples. Best of luck!

**Comment by radtkea** [comment]
I believe this is the [lectin](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/Isolectin_GS-IB4_AF568/0000-0003-2088-8310.html) Colin used to label endothelial cells of the human retina.

**Comment by rosedever** [comment]
Those are fantastic resources, thank you @radtkea! 

One resource that I have been using is the [NHP Reagents Reactivity](https://www.nhpreagents.org/ReactivityDatabase) page. I can look up the antibody before I buy it, to see if there is reported reactivity in each species, which has been helpful for me!

**Reply by radtkea** [reply]
Happy to help!

Thank you very much for sending the NHP Reagents Reactivity link. It's a very useful resource. Best of luck and please keep us posted:)
